numpy/ma/core.py

```diff
@@ -1157,9 +1135,8 @@ def __call__(self, a, b, *args, **kwargs):
         m |= getmask(a)
         m |= getmask(b)
         # Apply the domain
-        domain = ufunc_domain.get(self.f, None)
-        if domain is not None:
-            m |= domain(da, db)
+        if self.domain is not None:
+            m |= self.domain(da, db)
```

Comment:
The one actual code change. We don't need to look up `domain` in `ufunc_domain`, because we already know it